wikitongues/wikitongues/data_store/airtable/offset_utility.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+
+class OffsetUtility:
+    """
+    OffsetReader object contains offset value from file.
+    """
+
+    def __init__(self):
+        """
+        Construct OffsetReader
+        """
+        self.offset = None
+
+    @staticmethod
+    def read_offset():
+        """
+        Reads offset value from file
+
+        Returns:
+            offset (str): offset value
+        """
+        if os.path.exists(os.path.expanduser('~/.language-indexing-offset')):
```

Comment:
It would be good to define a constant for the filename string, since we have the same string 3 times. The constant can be declared at the top of the file. A good example to follow is in [airtable_item_extractor.py](https://github.com/wikitongues/Language-Indexing/blob/3c72981f8321e4ddfe5b6b2be61eddca7dc571a3/wikitongues/wikitongues/data_store/airtable/airtable_item_extractor.py#L7)